modules/manage/pages/iceberg/topic-iceberg-integration.adoc

```diff
@@ -49,7 +49,7 @@ In the Iceberg specification, tables consist of the following layers:
 ** Metadata file: Stores metadata about the table, including its schema, partition information, and snapshots. Whenever a change is made to the table, a new metadata file is created and becomes the latest version of the metadata in the catalog.
 --
 +
-In the Redpanda Iceberg integration, the manifest files are in JSON format.
+For Iceberg-enabled topics, the manifest files are in JSON format.
```

Comment:
Some general comments about this page: 

- I'd move the Iceberg concepts section to the top, above Prerequisites.
- For heading consistency, I'd rename "Schema support and mapping" to "Configure schema support and mapping"
- I'd introduce the prereqs similar to [this](https://docs.redpanda.com/current/manage/kubernetes/k-manage-topics/#prerequisites); e.g.,
- **rpk**: See Install or Update rpk
- **Enterprise license**: To check...
- **Tiered Storage**: To enable...

Reply:
Also, the image on this page seems extra large

Reply:
Updated per comments, although image resize is still pending